crypto/kzg/src/lib.rs

```diff
@@ -51,18 +51,41 @@ impl From<c_kzg::Error> for Error {
 #[derive(Debug)]
 pub struct Kzg {
     trusted_setup: KzgSettings,
-    context: Option<DASContext>,
+    context: DASContext,
```

Comment:
I guess we could keep this DASContext optional for the pre-peerDAS case, but then again if there's no perf impact maybe it's OK to keep it initialized

Reply:
there was no noticeable performance impact that I noticed (test-wise at least)

Reply:
This was changed to optional because it adds 550ms to startup time and 110 mb extra memory usage for an unused feature on mainnet:
https://github.com/sigp/lighthouse/pull/6212#discussion_r1712525802

It may also impact our CI time as I imagine Kzg is loaded many times.

A profile run on our basic-sim test show that 88% of CPU time is spent on loading `DASContext` during startup 
<img width="1809" alt="image" src="https://github.com/user-attachments/assets/513ae383-08fc-4a56-9d6a-78a5e1636f86">

I think we should make this optional until PeerDAS is scheduled.